pages/osx/defaults.md

```diff
@@ -0,0 +1,23 @@
+# defaults
+
+> Access OS X user defaults
+
+- read system default value
+
+`defaults read {{domain}} {{key}}`
+
+- read default values of applications
+
+`defaults read -app {{app_name}} {{key}}`
+
+- write key value
+
+`defaults write {{domain}} {{key}} {{-type}} {{value}}`
+
+- Speed up Mission Control animations
```

Comment:
This is a very specific example which I feel doesn't fit with the purpose of tldr and with the rest of the examples on this page. I'd suggest dropping it.